src/screens/InspectionCreate/index.jsx

```diff
@@ -3,23 +3,21 @@ import React, { useCallback } from 'react';
 import { CameraView } from '@monkvision/react-native-views';
 import { useNavigation } from '@react-navigation/native';
 
-import { TUTORIAL } from 'screens/names';
+import { GET_START, INSPECTION_REVIEW } from 'screens/names';
```

Comment:
GET_START isn't the greatest name in the world ;) Is it a "getting started" screen?